src/main/app/src/app/zaken/zaken.service.ts

```diff
@@ -176,68 +152,48 @@ export class ZakenService {
     medewerker?: GeneratedType<"RestUser">,
     reden?: string,
   ) {
-    return this.zacHttpClient
-      .PUT("/rest/zaken/lijst/verdelen", {
-        uuids,
-        groepId: groep?.id ?? "", // TODO: check generated interface
-        reden,
-        screenEventResourceId,
-        behandelaarGebruikersnaam: medewerker?.id,
-      })
-      .pipe(
-        catchError((err) => this.foutAfhandelingService.foutAfhandelen(err)),
-      );
+    return this.zacHttpClient.PUT("/rest/zaken/lijst/verdelen", {
+      uuids,
+      groepId: groep?.id ?? "", // TODO: check generated interface
+      reden,
+      screenEventResourceId,
+      behandelaarGebruikersnaam: medewerker?.id,
+    });
   }
 
   vrijgevenVanuitLijst(uuids: string[], reden?: string): Observable<void> {
-    return this.zacHttpClient
-      .PUT("/rest/zaken/lijst/vrijgeven", {
-        uuids,
-        reden,
-      })
-      .pipe(
-        catchError((err) => this.foutAfhandelingService.foutAfhandelen(err)),
-      );
+    return this.zacHttpClient.PUT("/rest/zaken/lijst/vrijgeven", {
+      uuids,
+      reden,
+    });
   }
 
   toekennenAanIngelogdeMedewerker(zaakUUID: string, reden?: string) {
-    return this.zacHttpClient
-      .PUT("/rest/zaken/toekennen/mij", {
-        zaakUUID,
-        reden,
-      })
-      .pipe(
-        catchError((err) => this.foutAfhandelingService.foutAfhandelen(err)),
-      );
+    return this.zacHttpClient.PUT("/rest/zaken/toekennen/mij", {
+      zaakUUID,
+      reden,
+    });
   }
 
   updateInitiator(
     zaak: GeneratedType<"RestZaak">,
     initiator: GeneratedType<"RestPersoon">,
     reason?: string,
   ) {
-    return this.zacHttpClient
-      .PATCH("/rest/zaken/initiator", {
-        zaakUUID: zaak.uuid,
-        betrokkeneIdentificatieType: initiator.identificatieType!,
-        betrokkeneIdentificatie: initiator.identificatie!,
-        roltypeUUID: undefined as unknown as string, // TODO: check this interface
-        roltoelichting: reason,
-      })
-      .pipe(
-        catchError((err) => this.foutAfhandelingService.foutAfhandelen(err)),
-      );
+    return this.zacHttpClient.PATCH("/rest/zaken/initiator", {
+      zaakUUID: zaak.uuid,
+      betrokkeneIdentificatieType: initiator.identificatieType!,
+      betrokkeneIdentificatie: initiator.identificatie!,
+      roltypeUUID: undefined as unknown as string, // TODO: check this interface
```

Comment:
Interesting TODO.. Not for this PR maybe but I am really against any TODOs in the code base.. TODO's belong in JIRA.